src/web/services/student.service.ts

```diff
@@ -124,4 +127,14 @@ export class StudentService {
     const responseType: string = 'text';
     return this.httpRequestService.get(ResourceEndpoints.STUDENTS_CSV, paramsMap, responseType);
   }
+
+  /**
+   * Returns a list of students enrolled in a course.
+   */
+  getStudentsEnrolledInCourse(queryParams: { courseId: string }): Observable<StudentListResults> {
+    const paramsMap: Record<string, string> = {
+      courseid: queryParams.courseId,
+    };
+    return this.httpRequestService.get(ResourceEndpoints.STUDENTS, paramsMap);
```

Comment:
A note, the data type returned by `/students` from the backend is not of the type `StudentListResults`.